contracts/infrastructure/ens/ArgentENSResolver.sol

```diff
@@ -46,7 +46,7 @@ contract ArgentENSResolver is Owned, Managed, ENSResolver {
      * @param _node The node to update.
      * @param _addr The address to set.
      */
-    function setAddr(bytes32 _node, address _addr) public onlyManager {
+    function setAddr(bytes32 _node, address _addr) external onlyManager {
```

Comment:
Same comment as above. Are these changes necessary for Slither or is just cleaning?